manifests/install.pp

```diff
@@ -2,5 +2,7 @@
 class nm::install {
   assert_private()
 
-  ensure_packages(['NetworkManager'])
+  package { 'NetworkManager':
```

Comment:
What is the purpose of this change?

Reply:
As there is actually no need, I have rolled back this change.